rules/resources.bzl

```diff
@@ -1687,13 +1688,13 @@ def _process_starlark(
         resources_ctx[_R_JAVA] = None
         resources_ctx[_PROVIDERS] = []
 
-    # TODO(b/69552500): In the Starlark Android Rules, the R compile time
-    # JavaInfo is added as a runtime dependency to the JavaInfo. Stop
-    # adding the R.jar as a runtime dependency.
+    r_java = None
+    if resource_files:
```

Comment:
Is there any reason why `if resource_files` shouldn't be `if resources_ctx[_R_JAVA]`, which was the past behavior on line 1696?